test/service/test_manifest.py

```diff
@@ -444,7 +444,8 @@ def test_manifest_zarr(self):
                         # Slice out file_prefix; -1 slice out postfix double quotes char
                         file_name = line[len(file_prefix):-1]
                         curl_files.append(file_name)
-                self.assertEqual([f['file_name'] for f in expected], curl_files)
+                self.assertEqual(sorted(freeze([f['file_name'] for f in expected])),
```

Comment:
I don't think freezing is necessary. Why do you use it?

Reply:
As mentioned during stand up, it was a misconception from my part that `freeze` was the standard for this type of situations.